### PR TITLE
update webhooks guides for dashboard

### PR DESCRIPTION
**Pull request by niall-maloney**

### CODE REVIEW

File: docs/webhooks/Getting-Started.md [hunk 1]
```diff
@@ -10,21 +10,21 @@ To test out webhooks locally without deploying an application to an environment
...
 Download ngrok [here](https://ngrok.com/download), then follow their getting started guide.
 
-## Configuring webhooks through the portal
+## Configuring webhooks through the dashboard
 
-Once you’re up and running with ngrok you will need to configure your webhooks through our portal, using the left menu to navigate to the webhook pages
+Once you’re up and running with ngrok you will need to configure your webhooks through our dashboard, using the left menu to navigate to the webhook pages
 
-1. Navigate to the webhooks secret page. Click on 'Generate secret' and save its value for later. Note that you may only have two active secrets.
+1. Navigate to the webhooks configuration section and click on the `+` button. Select your desired secret lifetime and then click on 'Create secret' and save its value for later. Note that you may only have two active secrets.
```
Comment (jenieto) [inline docs/webhooks/Getting-Started.md:17]:
```suggestion
1. Navigate to the webhooks configuration section and click on the `+` button in the secrets box/section/tile. Select your desired secret lifetime and then click on 'Create secret' and save its value for later. Note that you may only have two active secrets.
```
it's not fully clear where the '+' button is imo

**Review by jenieto — APPROVED** [review]

File: docs/webhooks/Getting-Started.md (hunk 1 above)
Comment (niall-maloney) [reply docs/webhooks/Getting-Started.md:17]: That card in the UI is called `Configuration`, secrets is a sub-heading. Would `click on the '+' button next to Secrets heading` be clearer?

Comment (jenieto) [reply docs/webhooks/Getting-Started.md:17]: ah I see, yeah that would be clearer

File: docs/webhooks/Testing.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Testing
 
-To enable you to check you have set up your infrastructure correctly, we offer a test event type. This can be triggered via API or in our portal in the subcriptions page.
+To enable you to check you have set up your infrastructure correctly, we offer a test event type. This can be triggered via API or in our dashboard webhooks configuration page.
 
-  ![Create a webhook subscription](../../assets/images/webhooks/Create-Subscription1.png)
+  ![Create a webhook subscription](../../assets/images/webhooks/Test-Event2.png)
```
Comment (bash-wealthkernel) [inline docs/webhooks/Testing.md:5]:
```suggestion
  ![Send test event](../../assets/images/webhooks/Test-Event2.png)
```

File: docs/webhooks/Testing.md [hunk 3]
```diff
@@ -19,22 +19,22 @@ It does not contain any information pertaining to a resource, just the following
...
+  "event": "This event was sent from WealthKernel Dashboard."
 }
 ```
 
 
-You can view the status of webhook requests in our portal.
+You can view the status of webhook requests in our dashboard.
 
-  ![Create a webhook subscription](../../assets/images/webhooks/Webhook-Messages.png)
+  ![Create a webhook subscription](../../assets/images/webhooks/Webhook-Messages2.png)
```
Comment (bash-wealthkernel) [inline docs/webhooks/Testing.md:33]:
```suggestion
  ![Messages list](../../assets/images/webhooks/Webhook-Messages2.png)
```

File: docs/webhooks/Testing.md [hunk 4]
```diff
@@ -19,22 +19,22 @@ It does not contain any information pertaining to a resource, just the following
...
 
-The details of an individual request can be seen from the details page, including the HTTP status code we received. You can also manually retry from this page should the automatic retries have been exhausted.
+The details of an individual request can be seen from the details page, including the HTTP status code we received. You can also manually resend from this page should the automatic retries have been exhausted.
 
-  ![Create a webhook subscription](../../assets/images/webhooks/Webhook-Messages-Details.png)
+  ![Create a webhook subscription](../../assets/images/webhooks/Webhook-Messages-Details2.png)
```
Comment (bash-wealthkernel) [inline docs/webhooks/Testing.md:37]:
```suggestion
  ![Message details](../../assets/images/webhooks/Webhook-Messages-Details2.png)
```

**Pull request merged**